Tests/GRPCHTTP2CoreTests/GRPCStreamStateMachineTests.swift

```diff
@@ -1343,8 +1343,36 @@ final class GRPCStreamServerStateMachineTests: XCTestCase {
   }
 
   func testSendMetadataWhenClientOpenAndServerIdle() throws {
-    var stateMachine = self.makeServerStateMachine(targetState: .clientOpenServerIdle)
-    XCTAssertNoThrow(try stateMachine.send(metadata: .init()))
+    var stateMachine = self.makeServerStateMachine(
+      targetState: .clientOpenServerIdle,
+      compressionEnabled: false
+    )
+    XCTAssertEqual(
+      try stateMachine.send(metadata: .init()),
+      [
+        ":status": "200",
+        "content-type": "application/grpc",
+        "grpc-accept-encoding": "deflate",
+      ]
+    )
+  }
```

Comment:
This is the server sending initial metadata back to the client, right? If so why do we get 'grpc-accept-encoding' when `compressionEnabled: false`?

Reply:
Ah, nice catch